kyo-core/shared/src/main/scala/kyo/Fiber.scala

```diff
@@ -598,10 +574,10 @@ object Fiber extends FiberPlatformSpecific:
             def onComplete(f: Result[E, A] => Unit)(using AllowUnsafe): Unit                             = self.onComplete(f)
             def onInterrupt(f: Result.Error[E] => Unit)(using Frame): Unit                               = self.onInterrupt(f)
             def block(deadline: Clock.Deadline.Unsafe)(using AllowUnsafe, Frame): Result[E | Timeout, A] = self.block(deadline)
-            def interrupt()(using frame: Frame, allow: AllowUnsafe): Boolean = self.interrupt(Panic(Interrupted(frame)))
-            def interrupt(error: Panic)(using AllowUnsafe): Boolean          = self.interrupt(error)
-            def interruptDiscard(error: Panic)(using AllowUnsafe): Unit      = discard(self.interrupt(error))
-            def mask()(using AllowUnsafe): Unsafe[E, A]                      = self.mask()
+            def interrupt()(using frame: Frame, allow: AllowUnsafe): Boolean      = self.interrupt(Panic(Interrupted(frame)))
+            def interrupt(error: Result.Error[E])(using AllowUnsafe): Boolean     = self.interrupt(error)
+            def interruptDiscard(error: Result.Error[E])(using AllowUnsafe): Unit = discard(self.interrupt(error))
```

Comment:
This is an unrelated change. `IOPromise` was changed some time ago to support `Error` interruptions instead of just `Panic` but the change wasn't reflected here.